### ISSUE TITLE
New stylesheet headerbar back button lacks arrow 

### ISSUE THREAD

**Issue by hanaral** [elementary/stylesheet#692]
Old stylesheet:
![image](https://user-images.githubusercontent.com/58219504/81948463-ad977100-95f9-11ea-80b2-f42a05a4aec5.png)
New stylesheet:
![image](https://user-images.githubusercontent.com/58219504/81948570-ca33a900-95f9-11ea-82c0-6ac214f9eb7c.png)

**Comment by cassidyjames** [comment]
Yep, as expected at this stage. The arrow style we did before has to be provided as an SVG that perfectly matches the final GtkCSS styling, and since the latter is in flux, it doesn't make sense to try and keep up with a hand-crafted SVG. Ideally we could use Granite to actually custom draw a back button, but I'm not sure if that's the concrete plan or not.

**Comment by hanaral** [comment]
Alright, I assumed the SVG wasn't done yet (I will admit I *did* fear that it might be going away)
